lib/fastlane/plugin/revenuecat_internal/version.rb

```diff
@@ -0,0 +1,5 @@
+module Fastlane
+  module RevenuecatInternal
+    VERSION = "0.1.0"
```

Comment:
Since this is internal, I think we can keep it like this for now. My idea is to import this plugin in the SDKs using github directly instead of RubyGems, but I'm open for discussion.

Reply:
I think that makes sense. If we made it public, it feels like RubyGems would make more sense then.